tests/CMakeLists.txt

```
@@ -43,6 +44,11 @@ MACRO(ADD_TESTCASE NAME GENERATED)
 
   PKG_CONFIG_USE_DEPENDENCY(${NAME} tinyxml)
   TARGET_LINK_LIBRARIES(${NAME} ${PROJECT_NAME})
+  if(RT_LIB)
+    TARGET_LINK_LIBRARIES(${NAME} ${PROJECT_NAME} ${RT_LIB})
+  else()
+    TARGET_LINK_LIBRARIES(${NAME} ${PROJECT_NAME} )
+endif()
```

```suggestion
  if(RT_LIB)
    TARGET_LINK_LIBRARIES(${NAME} ${RT_LIB})
  endif()
```
link from PROJECT_NAME to NAME is already done one line before